src/commands/org/api.ts

```diff
@@ -60,10 +62,17 @@ export class OrgApi extends SfCommand<void> {
       char: 'H',
       multiple: true,
     }),
+    'stream-to-file': Flags.string({
+      summary: 'Stream response to file',
+      helpValue: 'Example: report.xlsx',
+      char: 'S',
+      exclusive: ['include'],
+    }),
```

Comment:
If you have a better name for this flag, let me know. The `S` character is used by `curl` for "show error"